### ISSUE TITLE
Gibbing someone wearing shoes without the `DROP_ITEMS` flag set will still leave behind the shoes

### ISSUE THREAD

**Issue by MrMelbert** [tgstation/tgstation#79830]
## Reproduction:

- spawn a human with shoes
- right click -> smite -> gib
- all of their equipment will be deleted, with the exception of their shoes. 

I presume this is order of operations, removing the legs -> drop shoes -> delete mob -> delete stuff. It might even be the same with gloves.

**Comment by ATH1909** [comment]
![image](https://github.com/tgstation/tgstation/assets/42606352/87ab27f7-6bff-42dd-8590-c6ab1d0abe93)

**Comment by mc-oofert** [comment]
i think thats pretty funny and shouldnt even be considered a bug

**Comment by MrMelbert** [comment]
It is funny but if you want a gib to leave 0 items, it should leave 0 items. 
Otherwise there should be a flag for `leave 0 items except for shoes`.

**Comment by Sealed101** [comment]
yeah clothing deletion is handled by just qdel'ing the mob and it's triggered after the mob loses their limbs via `spread_bodyparts()`, which in turn does `drop_limb()` to handle gloves/shoes unequip. just passing an extra argument into there could work but it feels hacky as shit
https://github.com/tgstation/tgstation/blob/f59e23591e6f7f3e3cc046f52a36e734b3e5f483/code/modules/mob/living/carbon/death.dm#L20-L31
https://github.com/tgstation/tgstation/blob/f59e23591e6f7f3e3cc046f52a36e734b3e5f483/code/modules/mob/living/death.dm#L11-L28
https://github.com/tgstation/tgstation/blob/f59e23591e6f7f3e3cc046f52a36e734b3e5f483/code/modules/surgery/bodyparts/dismemberment.dm#L198-L230
